Behavioral.Automation.Bindings.UI.Tests/Configurations/WebElement.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Behavioral.Automation.Bindings.UI.Abstractions;
+using Behavioral.Automation.Bindings.UI.Context;
+
+namespace Behavioral.Automation.Bindings.UI.Tests.Configurations;
+
+public class WebElement : IWebElement, IButtonElement
+{
+    public WebContext WebContext { get; }
+    public ElementSelector ElementSelector { get; }
+    public string? Description { get; set; }
+    public Task ClickAsync()
+    {
+        return Task.Run(() => Console.WriteLine("Clicked button"));
```

Comment:
I understand that it's for showing purposes, but I don't think it's a good idea to run Console in separate thread. It's a shared resource, so should not be really thread safe.
Why not, just use async/await and if you want something to await for, just do - [await Task.Delay(50)] before Console.Writeline().

Reply:
It's more for testing configuration, not demonstration. We can set up CI/CD check if someone tries to update bindings.

ClickAsync() method should return Task. Instance of Task was created by running Run method with some random action.

Reply:
Changed to return Task.Delay(50)

Thank you very much!

Reply:
@Oleg-Mozhey to output directly to test output you can do this:
```
public class ClickBinding
{
    //Interface for interacting with test output
    private readonly ISpecFlowOutputHelper _outputHelper;

    //Constructor injection will resolve default ISpecFlowOutputHelper implementation without implicit registration
    public ClickBinding(ISpecFlowOutputHelper outputHelper)
    {
        _outputHelper = outputHelper;
    }
    
    [When("user says \"(.*)\"")]
    public async Task ShowMessage(string message)
    {
        await Task.Run(() => _outputHelper.WriteLine(message));
    }
 }
```
@LukaRukhadze there should not be any issues with using Console methods in Tasks, [as per documentation](https://learn.microsoft.com/en-us/dotnet/api/system.console?view=net-8.0&redirectedfrom=MSDN#thread-safety)

Reply:
@vladimir-s-quantori Great suggestion, thanks a lot!
I'll add this in future PRs covering interfaces implementation.

For simple test case any method is enough. I don't actually need to print something in Console. The idea is to register implementation in Behavioral.Automation.Bindings.UI.Test and check that SpecFlow can understand this and use the implementation I want (no errors are thrown)